tests/conftest.py

```
@@ -0,0 +1,44 @@
+        assert mock_version.major == 3
+        assert mock_version < (3, 13)
+        assert mock_version[1] == 12
+    """
+
+    def __init__(self, major: int, minor: int, micro: int):
+        self.major = major
+        self.minor = minor
+        self.micro = micro
+        self._tuple = (major, minor, micro)
+
+    def __lt__(self, other):
```

Ruff (0.14.14)

[warning] Missing return type annotation for special method `__lt__`

(ANN204)